lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -7,8 +7,16 @@ module Fastlane
 
   module Helper
     class GithubHelper
+      def self.github_token
+        [
+          'GHHELPER_ACCESS', # For historical reasons / backward compatibility
+          'GITHUB_TOKEN',    # Used by the `gh` CLI tool
+        ].map { |key| ENV[key] }
+          .detect { |value| !value.nil? }
```

Comment:
If I read this correctly, in case where the environment has both `GHHELPER_ACCESS` and `GITHUB_TOKEN`, the code will pick the former, legacy one. 

I think the desired resolution would be to pick the modern value, right?

I mean, it doesn't change much because both work, so maybe 🤷

Reply:
@mokagio I asked the same question [here](https://github.com/wordpress-mobile/release-toolkit/pull/313#discussion_r758538199) and Jeremy made a good point that while this would be nice, this would be a breaking change to do so…